bin/tessel-2.js

```diff
@@ -144,6 +144,15 @@ parser.command('provision')
   })
   .help('Authorize your computer to control the USB-connected Tessel');
 
+parser.command('unbrick')
+  .callback(callControllerCallback('unbrickTessel'))
+  .option('openwrt', {
+    abbr: 'o',
+    flag: true,
+    help: 'Restore a factory version of OpenWrt'
+  })
+  .help('Unbrick your Tessel by restoring to factory settings.');
+
```

Comment:
@johnnyman727 let me know if you would like to rename all occurrences of unbrick to restore.